front/src/modules/settings/data-model/components/SettingsObjectFieldPreview.tsx

```diff
@@ -145,7 +146,11 @@ export const SettingsObjectFieldPreview = ({
             hotkeyScope: 'field-preview',
           }}
         >
-          <FieldDisplay />
+          {fieldType === 'BOOLEAN' ? (
+            <BooleanFieldInput readonly />
```

Comment:
I think we should avoid introducing edge cases and treat it for all FieldInputs, either:
- introduce an optional readonly in FieldContext (but that's discutable, as as readonly state might be a bit lighter in the future)
- can we add a onClick on StyledFieldPreview that will prevent event propagation and capture the click so it does not trigger the boolean?

Reply:
Discussed with you, it's better to leave it like this for now because:
- readonly <> disabled
- we don't want to add an extra layer to wrap these Display/Input for now